docs/android.md

```diff
@@ -0,0 +1,17 @@
+# Instructions For Android Phone Using UserLand 
+📱-Install Userland From Playstore First  
+📱-Install Ubuntu VM  
+📱-Start It  
+📱-Install NodeJS, Curl And Yarn Using `sudo apt install nodejs npm yarn curl -y`  
+📱-Install NVM :  
```

Comment:
Can we change this to a numbered list?

Reply:
Ofc

Reply:
Oh, I hope you don't mind me looping in... `sudo` actually doesn't exist on Termux, and I think it's kinda hard to install it since the `sudo` package(aka `tsu`) is kinda a problem. I can't try installing it without a mirror error. So, maybe we need to remove `sudo` in line 5.

Reply:
Sorry for that...

Reply:
Awesome! Let me know after you fix it and then we can get this merged!

Reply:
It’s not termux
It’s userland on playstore